### PR TITLE
KNS_002: Added Migration Script to Insert Sales Employees

### PR DESCRIPTION
**Pull request by MatthewJKainos**

### CODE REVIEW

**Review by Aaron-Millen — APPROVED** [review]

File: src/main/resources/db/migration/V4__insert_sales_employees.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+INSERT INTO Employee (name, national_insurance_no, bank_account, salary) VALUES
+("Benji Agnew", "PL 22 34 54 B", "GB 31839 0923 9343 2394", 25000),
+("Delilah Martin", "PL 20 12 77 B", "GB 9872 4432 9663 2224", 30000);
+
+INSERT INTO Sales_Employee (employee_id) VALUES
+(1), (2);
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V4__insert_sales_employees.sql]: we shouldnt hardcode id's if they are being inserted

File: src/main/resources/db/migration/V4__insert_sales_employees.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+SELECT * FROM Employee;
+SELECT * FROM Sales_Employee;
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V4__insert_sales_employees.sql]: why are these here?

**Review by lukemKainos — APPROVED** [review]
